### DISCUSSION TITLE
Issue with nextTick()

### DISCUSSION THREAD

**Discussion by ggedde** [vuejs/petite-vue#31] [Q&A]
On a checkbox I am using `<input type="checkbox" v-model="bob" @change="checkBob()">`
Within checkBob() this.bob is false even though the checkbox is checked.

I tried the following `<input type="checkbox" v-model="bob" @change="$nextTick(checkBob)">`
But I get the same results.

However, within checkBob() if I add a setTimeout() for 0 delay then bob is true.
Not a big deal, but I would like to know if I am doing it the right way.
How do you properly use nextTick when using the CDN method with importing it?

Thanks

**Comment by nooooooom** [comment]
As far as I know, there is no distinction between the events on the `v-model` and the native events of the node in the functional implementation.
Look [here](https://github.com/vuejs/petite-vue/blob/63c60d34364ff0fa493dca1aba1407d849bdc3af/src/walk.ts#L67), the side effects of `v-mode` are later than other directives, so the change of `bob` after the `@change` .
If you use `@change` on the `v-model` node, you will still listen to native events. If you need to monitor the CheckBox value changes, you can use `$event.target.checked`. if you need to monitor the `bob` changes, you can use `v-effect="checkBob()"`.
